CHANGELOG.md

```diff
@@ -45,6 +45,9 @@
 
 # 0.49.0 - [unreleased]
 
+- Remove default features. You need to enable all features explicitly now. As a quick workaround, you may want to use the
```

Comment:
```suggestion
- Remove default features. You need to enable required features explicitly now. As a quick workaround, you may want to use the
```

I think otherwise folks would think that they just need to enable **all** features manually from now on.